src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -1280,7 +1280,81 @@ def eq():
 
     def equals():
         """
-        Test whether two objects contain the same elements.
+        Test whether two dataframes contain the same elements.
+
+        This function allows two DataFrames to be compared against
+        each other to see if they have the same shape and elements. NaNs in
+        the same location are considered equal.
+
+        The row/column index do not need to have the same type, as long
+        as the values are considered equal. Corresponding columns and
```

Comment:
> Corresponding columns and index must be of the same dtype.

In the documentation for the new query compiler method, you mention that Snowpark pandas won't enforce type equivalence within integer variants because of our type mapping rules. Can you call this out in the external-facing API documentation as well?